src/components/Select/styles.styl

```diff
@@ -0,0 +1,19 @@
+@require 'settings/palette'
+
+.Select
+    appearance none
```

Comment:
Did you test this on IE ? I remember it doesn't support `appearance`, so it renders the system arrow (see https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance). There are workarounds for IE : https://stackoverflow.com/questions/1895476/how-to-style-a-select-dropdown-with-css-only-without-javascript/13968900#13968900

Reply:
OK thanks !